mongoengine/queryset/base.py

```diff
@@ -1715,29 +1728,29 @@ def no_dereference(self):
 
     def _item_frequencies_map_reduce(self, field, normalize=False):
         map_func = """
-            function() {
-                var path = '{{~%(field)s}}'.split('.');
+            function() {{
```

Comment:
Can you elaborate on the changes in this method?

Reply:
This wasn't actually something I introduced - I believe it was modified/generated when I had to run the pre-commit hook.

If I removed the changes, it would simply fail...I can double check that on a fresh branch and see what happens...My memory of that is a little vague...but I explicitly remember not having made the change myself but having been forced to by something else.

Reply:
Not sure what happened with that method - but I was able to revert back to what was in the main branch 💪 

resolved!